deep_translator/google_trans.py

```diff
@@ -27,8 +27,19 @@ def __init__(self, source="auto", target="en", proxies=None, **kwargs):
         self.__base_url = BASE_URLS.get("GOOGLE_TRANSLATE")
         self.proxies = proxies
 
-        if self.is_language_supported(source, target):
-            self._source, self._target = self._map_language_to_code(source.lower(), target.lower())
+        # code snipppet that converts the language into lower-case and skip lower-case conversion for abbreviations
+        # since abbreviations like zh-CN if converted to lower-case will result into error
+        #######################################
+        source_map = source
```

Comment:
It should be noted that we can't lower some of the abbreviations like ```zh-CN or zh-TW``` since in the translator database they are defined like this, therefore it is mandatory to check whether the source or target are not abbreviations and if the condition satisfies then we can lower the ```source``` and ```target```. I just concatenated source and target with ```_map``` to assign it as a new name.

I would also like to point that earlier in the code as shown  
```
 if self.is_language_supported(source, target):
            self._source, self._target = self._map_language_to_code(source.lower(), target.lower())
```

First-line was not passed with (lower) function which was causing the issue of case sensitivity for example ```English``` was discarded and raises the exception of language not supported.

PS: I assigned source and target with name 
```
source_lower
target_lower
```
for better clarity and readibilty.

Reply:
If it is clear can I mark this as resolved?

Reply:
@kuspia yes